stdlib/public/Concurrency/TaskPrivate.h

```diff
@@ -713,7 +738,7 @@ inline void AsyncTask::flagAsEnqueuedOnExecutor(ExecutorRef newExecutor) {
 
 inline void AsyncTask::flagAsSuspended() {
   SWIFT_TASK_DEBUG_LOG("%p->flagAsSuspended()", this);
```

Comment:
I could do with inputs on naming here. In https://github.com/apple/swift/pull/41088, John suggested that we rename the following methods on the task (`flagAsSuspended`, `flagAsRunning`, `flagAsCompleted`, `flagAsEnqueuedOnExecutor`) since they are doing more than bookkeeping at this point. 

A few suggestions for common prefix names that come to mind:

`make-<enqueued,running,suspended>`
`flagAsAnd<enqueue,run,suspend>`
`do<enqueue,run,suspend>`